packages/opentelemetry/lib/tracing/exporter.js

```diff
@@ -0,0 +1,31 @@
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import { ZipkinExporter } from '@opentelemetry/exporter-zipkin';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+import { ConsoleSpanExporter } from '@opentelemetry/tracing';
+
+const DEFAULT_SERVICE_NAME = 'appium';
+
+const AVAILABLE_EXPORTERS = {
+  JAGGER: 'jagger',
+  ZIPKIN: 'zipkin',
+  PROMETHEUS: 'prometheus',
+  CONSOLE: 'console'
+};
+
+function buildExporter (exporterType, config = null) {
+  switch (exporterType) {
+    case AVAILABLE_EXPORTERS.JAGGER:
+      config = config || { serviceName: DEFAULT_SERVICE_NAME };
```

Comment:
do we really need to reassign `config` arg? `return new JaegerExporter(config || { serviceName: DEFAULT_SERVICE_NAME })` should be ok